### ISSUE TITLE
Unity 5.4.0f3 and LR shading issue

### ISSUE THREAD

**Issue by steelwoolstudios** [ValveSoftware/the_lab_renderer#9]
I just switched our project to 5.4.0f3 and installed Lab Renderer. We are using Standard Unity shaders and when I do a convert, I get to errors in the console but the objects become invisible. I can convert back and it is fine.

![convert1](https://cloud.githubusercontent.com/assets/7539994/17784543/c281ace0-6531-11e6-8856-b3b50c80514f.png)

![convert2](https://cloud.githubusercontent.com/assets/7539994/17784547/c66742ac-6531-11e6-8844-5dfbd42a61c6.png)

**Comment by SmartCarrion** [comment]
Do you have the valve camera component on the right camera? And make sure 'hide all valve materials' is not checked.

**Comment by steelwoolstudios** [comment]
Shit. I do now! Thanks!

**Comment by BHSPitMonkey** [comment]
Can you close this?

**Comment by Wip3ou7** [comment]
I'm running into this problem too, I'm using the valve example scene and it already has a VR camera and hand tracking rig in the scene from what I can tell. Where is this "hide all valve materials" checkbox?

**Comment by SmartCarrion** [comment]
It is on the ValveCamera script.  look for "Camera (eye)" in the heirarchy

**Comment by BHSPitMonkey** [comment]
@Wip3ou7 yes, you already have a VR camera, but that camera object needs to have a ValveCamera component added to it (you need to do this yourself). The checkbox you asked about is on that component, though it won't be checked by default so you shouldn't have to worry about it.
